redis.services.yml

```diff
@@ -4,3 +4,6 @@ services:
     arguments: ['@redis.factory', '@cache_tags.invalidator.checksum']
   redis.factory:
     class: Drupal\redis\ClientFactory
+  redis.phpredis.invalidator:
+    class: Drupal\redis\Cache\RedisCacheTagsChecksum
+    arguments: ['@redis.factory']
```

Comment:
I am not really sure why I wrote this, I had trouble registering the checksum invalidator into services, @Berdir please tell me if this right or if I need to remove it.